tests/unit/core/components/TestInstallButton.js

```diff
@@ -34,11 +34,13 @@ describe('<InstallButton />', () => {
     getClientCompatibility: () => ({ compatible: true }),
     hasAddonManager: true,
     i18n: getFakeI18nInst(),
+    store: createStore().store,
     ...customProps,
   });
 
-  const render = (props) =>
-    shallow(<InstallButtonBase {...renderProps(props)} />);
+  const render = (props) => shallowUntilTarget(
+    <InstallButton {...renderProps(props)} />, InstallButtonBase
```

Comment:
Another option that I never took seriously:

* We could put a special property on each HOC so that the unwrapper can pop off each one automatically based on presence of the property. This feels dirty and will be error prone since you'd have to remember to decorate each HOC with the special property.